coredns/README.md

```diff
@@ -2,20 +2,21 @@
 
 ## Overview
 
-Get metrics from CoreDNS in real time to visualize and monitor DNS failures and cache hits/misses.
+Get metrics from CoreDNS in real time to visualize and monitor DNS failures and cache hits or misses.
 
 ## Setup
 
-### Installation
+### Prerequisites
 
-The CoreDNS check is included in the [Datadog Agent][1] package, so you don't need to install anything else on your servers.
+This OpenMetrics-based integration has a latest version (V2) and a legacy version (V1). To get all the most up-to-date features, Datadog recommends upgrading to the latest version. For more information, see the [Latest and Legacy Versioning For OpenMetrics-based Integrations][21].
 
-**Note**: The current version of the check (1.11.0+) uses [OpenMetrics][17] (OpenMetricsBaseCheckV2) for metric collection, which requires Python 3.
-For hosts unable to use Python 3, or if you previously implemented this integration version, see the [legacy OpenMetricsBaseCheckV1 example][18].
-There is an exception to that for Autodiscovery users relying on the `coredns.d/auto_conf.yaml` file, which enables the `prometheus_url` option for the OpenMetricsBaseCheckV1 legacy version of the check by default.
-See the sample [coredns.d/auto_conf.yaml][19] for the default configuration options and the sample [coredns.d/conf.yaml.example][20] for all available configuration options.
+The CoreDNS check version 1.11.0 or later uses [OpenMetrics][17]—otherwise known as the latest version (OpenMetricsBaseCheckV2)—for metric collection, which requires Python 3. The `latest` version of the CoreDNS check submits `.bucket` metrics and submits the `.sum` and `.count` histogram samples as monotonic count type. These metrics were previously submitted as `gauge` type in the legacy version OpenMetricsBaseCheckV1. See the [`metadata.csv` file][14] for a list of metrics available in each version. 
```

Comment:
```suggestion
The CoreDNS check version 1.11.0 or later uses [OpenMetrics][17]-otherwise known as the latest version (OpenMetricsBaseCheckV2)-for metric collection, which requires Python 3. The `latest` version of the CoreDNS check submits `.bucket` metrics and submits the `.sum` and `.count` histogram samples as monotonic count type. These metrics were previously submitted as `gauge` type in the legacy version OpenMetricsBaseCheckV1. See the [`metadata.csv` file][14] for a list of metrics available in each version. 
```

To fix the validation job